randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -50,13 +50,15 @@ Extra - room_id: [0]
           Screw Attack
           Any of the following:
               # Break single bomb blocks
-              Can Use Power Bombs
               Screw Attack and Knowledge (Beginner)
-              # TODO: vid of this
+              # Jump Bomb Jump to break the blocks: https://youtu.be/mZe9Ht3o19w
               Hi-Jump and Jump Bombjump (Expert) and Can Use Bombs
+              # Break the blocks with Power Bombs: https://youtu.be/A89giARXvPE
+              Can Use Power Bombs
           Any of the following:
               # Climb up
```

Comment:
Your video above made me realize: couldnt you also climb up in specific situation freeze the owls to go up?

Reply:
I think theoretically yes if the power bomb does not kill all of the Menus in the room and you can lure them over to the newfound hole in the ground. You could probably lay the power bomb early on the left side so the horizontal radius limits the damage to enemies.

Reply:
even if it doesnt work with the PB, you could still use it as an alternative to breaking the blocks with screw